src/datepicker/datepicker-day-view.ts

```diff
@@ -0,0 +1,20 @@
+import {Component, Input} from '@angular/core';
+import {MonthViewModel, DayViewModel} from './datepicker-view-model';
+
+@Component({
+  selector: '[ngb-datepicker-day-view]',
+  styles: [`
+    :host {      
+      text-align: center;
+      padding: 0.185rem 0.25rem;      
+      border-radius: 0.25rem;
+    }
+  `],
+  host: {'[class.bg-primary]': 'selected', '[class.text-muted]': 'day.date.month !== month.number || day.disabled'},
```

Comment:
How about extracting `day.date.month !== month.number || day.disabled` to a method (`isDisabled()`?). Probably would be easier to read / follow / understand.